pkg/build/api/v1/types.go

```diff
@@ -371,6 +375,13 @@ type SourceBuildStrategy struct {
 	ForcePull bool `json:"forcePull,omitempty" description:"forces the source build to pull the image if true"`
 }
 
+// BuildPostCommitSpec holds a build post commit hook specification.
+type BuildPostCommitSpec struct {
+	// RunBash holds a bash script to be executed in a container running the
+	// build output image.
+	RunBash string `json:"runBash,omitempty" description:"bash script to be executed in a container running the build output image"`
```

Comment:
@bparees before we move into that direction, I'd like to discuss how this will eventually be exposed to the end user.

1\. Using the CLI, how will an end user set "the command to run tests"?

During R&D, we've been using `oc edit`, which gives a suboptimal UX, and requires one to type the names of the fields correctly, with proper quoting, etc. Easy to make mistakes and lose your work, even though a small work.

2\. Same question as above but for the Web Console.

I had in mind a single text area just like other CIs offer. If we have a way to setup the entrypoint and cmd, and those are string slices, it's harder for me to imagine a good UI/UX.